calling/src/commonMain/kotlin/com/wire/kalium/calling/CallType.kt

```diff
@@ -0,0 +1,6 @@
+package com.wire.kalium.calling
+
+enum class CallType(val avsValue: Int) {
+    NORMAL(avsValue = 0),
```

Comment:
is the NORMAL the AUDIO call? or there is difference?

Reply:
Exactly 
```
NORMAL for video call
VIDEO for video call
```

Reply:
Thanks, So why we don't call it `Audio`?

Reply:
Good point..I just followed what 's already exist in AVS lib..But yeah I agree `AUDIO` is more clear